packages/decorators/tests/subcommands/chatinput.test.ts

```diff
@@ -0,0 +1,70 @@
+import { ChatInputSubcommand } from '../../src';
+
+describe('Test if chatinput subcommands do get mapped', () => {
+	test('Test for single chatInput subcommand', () => {
+		class Test {
+			@ChatInputSubcommand('test')
+			public hello() {
+				return 'Hello';
+			}
+		}
+
+		const instance = new Test();
+		expect(Reflect.get(instance, 'subcommandMappings')).toStrictEqual([
+			{
+				name: 'test',
+				type: 'method',
+				chatInputRun: 'hello'
+			}
+		]);
+	});
+	test('Test for multiple chatInput subcommands', () => {
```

Comment:
2 nits:

1 newline each method block

2 use `GIVEN x THEN y`  for test strings